examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+#
+# - Fractional anisotropy (FA),
+#
+# - Mean diffusivity (MD),
+#
+# - Axial diffusivity (AD),
+#
+# - and Radial diffusivity (RD)
+#
+# In this example we will only use FA.
+#
+# .. note::
+#   By default:
+#
+#   - All b-values less than or equal to 50 are considered to be
+#     without diffusion weighting.
+#
+#   - No binary masks are applied; therefore all voxels are processed.
+#
+# .. note::
+#   The diffusion tensor imaging parameters contain the associated eigenvalues
+#   and eigenvectors from eigen decomposition on the diffusion tensor.
+
+
+print("Calculating DTI...")
+
+if not op.exists(op.join(working_dir, 'dti_FA.nii.gz')):
+    dti_params = dti.fit_dti(hardi_fdata, hardi_fbval, hardi_fbvec,
+                             out_dir=working_dir)
+else:
+    dti_params = {'FA': op.join(working_dir, 'dti_FA.nii.gz'),
+                  'params': op.join(working_dir, 'dti_params.nii.gz')}
+
+print(f"Loading {dti_params['FA']}")
+FA_img = nib.load(dti_params['FA'])
+FA_data = FA_img.get_fdata()
+
+show_anatomical_slices(FA_data, 'Fractional Anisotropy (FA)')
+
+##########################################################################
+# Register the individual data to a template:
+# -------------------------------------------
+# For the purpose of bundle segmentation, the individual brain is registered
+# to the MNI T2 template. The waypoint ROIs used in segmentation are then each
+# brought into each subject's native space to test streamlines for whether they
+# fulfill the segmentation criteria.
+#
+# .. note::
+#    Here, for simplicity we only perform a non-linear transformation between
+#    the individual's brain and the MNI T2 template. This is likely why the
+#    occipital/parietal junction appears concave. In practice, it's a good idea
+#    to also perform a pre-alignment using an affine transformation. Moreover,
+#    by default, registration uses APM (Anisotropic Power Map) which also tends
+#    to provide better results.
+
+
+print("Registering to template...")
+
+MNI_T2_img = afd.read_mni_template()
+nib.save(MNI_T2_img, op.join(working_dir, 'MNI_T2.nii.gz'))
+show_anatomical_slices(MNI_T2_img.get_fdata(), 'MNI T2 Template')
+
+# create a diffeomorphic map from subject to MNI space
+if not op.exists(op.join(working_dir, 'mapping.nii.gz')):
+    import dipy.core.gradients as dpg
+    gtab = dpg.gradient_table(hardi_fbval, hardi_fbvec)
+
+    warped_hardi, mapping = reg.syn_register_dwi(hardi_fdata, gtab)
+    reg.write_mapping(mapping, op.join(working_dir, 'mapping.nii.gz'))
+else:
+    mapping = reg.read_mapping(op.join(working_dir, 'mapping.nii.gz'),
+                               img,
+                               MNI_T2_img)
+
+mapping_img = nib.load(op.join(working_dir, 'mapping.nii.gz'))
+mapping_img_data = mapping_img.get_fdata()
+
+# Working with diffeomorphic map data
+mapping_img_data = mapping_img_data[..., 0, 0]
+show_anatomical_slices(mapping_img_data, 'Registration Displacement Mapping')
```

Comment:
@arokem Is there a better more meaningful name for mapping file?

MNI T2 to Subject B0 Registration Symmetric Normalization Displacement Mapping